ISSUE_GUIDELINES.md

```diff
@@ -0,0 +1,44 @@
+# Issue Guidelines
+
+Issues filed in this repository are specific to the Kubecost Helm chart only. If your issue pertains to the Kubecost application and not the Helm chart, follow the guidance below.
+
+To help route you to the best location, see some examples of feature requests and bug reports [below](#examples-of-helm-requests).
+
+## Kubecost Feature Requests
+
+For all feature requests to Kubecost, please create an enhancement request in the [feature requests repository](https://github.com/kubecost/feature-requests).
+
+## Kubecost Bug Reports
+
+Bug reports for the Kubecost application stack should be directed to one of the following locations, both of which will result in creation of a support ticket.
```

Comment:
@dwbrown2 just double checking that this is expected for you - Kubecost applications bugs would have to come through email or slack.

Reply:
Features, yes. Bugs, no.

Reply:
@chipzoller I thought we landed allowing users to log bugs in the new repo? I feel like theres enough value in allowing users to file KC applications issues in GH to justify the extra maintenance effort.

Reply:
That's totally fine if you and @dwbrown2 want to go that route. The repo name needs to change in that case, but if you have a GitHub repo which accepts bugs and feature requests, you're now working in two different places. You're fine with that?

Reply:
Yea lets try it out - I can pull in more support as needed

Reply:
After we fix this section (new repo for Kubecost bugs, update `ISSUE_GUIDELINES.md` to reflect this), the rest of the PR LGTM!

Reply:
@kwombach12, please suggest an appropriate name for a new repository which will be the new home for application bugs and features.

Reply:
@kwombach12, please provide a repo name so this can move forward.